ui/widget.js

```diff
@@ -450,16 +450,12 @@ $.Widget.prototype = {
 			currentElements = this.classesElementLookup[ classKey ];
 			if ( value[ classKey ] === this.options.classes[ classKey ] ||
 					!currentElements ||
-					!currentElements.length ) {
+					!currentElements.size ) {
 				continue;
 			}
 
-			// We are doing this to create a new jQuery object because the _removeClass() call
-			// on the next line is going to destroy the reference to the current elements being
-			// tracked. We need to save a copy of this collection so that we can add the new classes
-			// below.
-			elements = $( currentElements.get() );
-			this._removeClass( currentElements, classKey );
+			elements = $( Array.from( currentElements ) );
```

Comment:
We re-wrap here, but the previous code was already creating a new jQuery object, so it shouldn't be worse than before.

Also, I don't understand the comment above; how would we lose the reference here? It's already in the `currentElement` object, that wouldn't just disappear...